Buildings/Applications/DHC/EnergyTransferStations/CoolingIndirect.mo

```diff
@@ -0,0 +1,287 @@
+within Buildings.Applications.DHC.EnergyTransferStations;
+model CoolingIndirect
+  "Indirect cooling energy transfer station for district energy systems"
+  extends Buildings.Fluid.Interfaces.PartialFourPort(
+    redeclare package Medium1 = Medium,
+    redeclare package Medium2 = Medium);
+
+  package Medium = Buildings.Media.Water;
+
+  parameter Modelica.SIunits.SpecificHeatCapacity cp=
+    Medium.specificHeatCapacityCp(
+      Medium.setState_pTX(Medium.p_default, Medium.T_default, Medium.X_default))
+    "Default specific heat capacity of medium";
+
+  // mass flow rates
+  parameter Modelica.SIunits.MassFlowRate m1_flow_nominal(min=0,start=0.5)
+    "Nominal mass flow rate of primary (district) district cooling side";
+  parameter Modelica.SIunits.MassFlowRate m2_flow_nominal(min=0,start=0.5)
+    "Nominal mass flow rate of secondary (building) district cooling side";
+
+  // Heat exchanger
+  parameter Modelica.SIunits.PressureDifference dp1_nominal(min=0,start=500)
+    "Nominal pressure difference on primary side"
+    annotation(Dialog(tab="Heat exchanger"));
+  parameter Modelica.SIunits.PressureDifference dp2_nominal(min=0,start=500)
+    "Nominal pressure difference on secondary side"
+    annotation(Dialog(tab="Heat exchanger"));
+  parameter Boolean use_Q_flow_nominal=true
+    "Set to true to specify Q_flow_nominal and temperatures, or to false to specify effectiveness"
+    annotation(Dialog(tab="Heat exchanger"));
+  parameter Modelica.SIunits.HeatFlowRate Q_flow_nominal(min=0,start=10000)
+    "Nominal heat transfer"
+    annotation(Dialog(tab="Heat exchanger"));
+  parameter Modelica.SIunits.Temperature T_a1_nominal(min=0+273,max=100+273,start=5+273)
```

Comment:
use `273.15` instead of `273`.

Reply:
Thanks for catching this. Corrected for next commit.